src/lib/style/components/_query-wrapper-plot-nav.scss

```diff
@@ -122,46 +122,6 @@
     margin-bottom: 10px;
   }
 
-  .TopLevelControls {
```

Comment:
Used to be only under the Query Wrapper plot nav, but it now might be shown above a SynapseTable (in StandaloneQueryWrapper)